src/orb.yml.hbs

```diff
@@ -171,6 +172,11 @@ jobs:
           AWS region to operate in. Defaulted to $AWS_REGION
         type: string
         default: $AWS_REGION
+      profile-name:
+        description:
+          AWS profile used for calls to aws api.
+        type: string
+        default: default
```

Comment:
Could we make it "" by default? Ideally, I would like `--profile` to not be set when the value of `profile-name` is `""`, to avoid conflicts with existing usage of the orb.

Reply:
Could you update `.circleci/config.yml` to add `profile-name: default` to the `aws-ecs/deploy-service-update` job at https://github.com/CircleCI-Public/aws-ecs-orb/blob/8626489833e615ba3c0e5b846c4c55d55095e5cd/.circleci/config.yml#L575 and https://github.com/CircleCI-Public/aws-ecs-orb/blob/8626489833e615ba3c0e5b846c4c55d55095e5cd/.circleci/config.yml#L806 to include the new parameter in our integration tests?